package.json

```diff
@@ -123,6 +125,8 @@
     "uuid": "7.0.3"
   },
   "peerDependencies": {
+    "@types/styled-components": "4.1.8",
+    "@types/warning": ">=3.0.0",
```

Comment:
I get the error that `Could not find a declaration file for module 'warning'`

So if this package is needed when using `"skipLibCheck": false` maybe add that also to the documentation?

Reply:
Yes, this is expected behavior (this is what peerDependencies mean - you need to handle them on your project and not included in the bundle of referring package.json). Will improve documentation for this.